hack/make-rules/helpers/cache_go_dirs.sh

```diff
@@ -44,10 +44,10 @@ function kfind() {
         \(                         \
         -not \(                    \
             \(                     \
+                -path ./\*/_\* -o  \
+                -path ./\*/.\* -o  \
```

Comment:
been a while since I've looked at find expressions. We don't have to do `**` to get for globbing, do we?

Reply:
Won't matter now, resolving